src/reanimated2/helperTypes.ts

```diff
@@ -123,8 +123,6 @@ export type AnimateProps<P extends object> = NonStyleAnimatedProps<P> &
     animatedProps?: Partial<AnimatedPropsProp<P>>;
   };
 
-export type AnimatedProps<P extends object> = AnimateProps<P>;
```

Comment:
Please restore this type and mark it with ts-prune. It's an alias for AnimateProps since AnimateProps is a terrible type name - I will use it in the future.